lib/this_feature/adapters/memory.rb

```diff
@@ -28,11 +28,7 @@ def on?(flag_name, context: nil, data: {})
       end
 
       def off?(flag_name, context: nil, data: {})
-        on_result = on?(flag_name, context: context)
-
-        return if on_result.nil?
-
-        !on_result
+        !on?(flag_name, context: context)
```

Comment:
```suggestion
        !on?(flag_name, context: context, data: data)
```

Reply:
Data isn't actually supported, but it doesn't hurt :) 
Good catch though!